tests/migration_test.go

```diff
@@ -1462,6 +1390,86 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 		})
 
 		Context("migration postcopy", func() {
+
+			var (
+				pvName string
+			)
+
+			BeforeEach(func() {
+				pvName = "test-nfs" + rand.String(48)
+				setMigrationBandwidthLimitation(resource.MustParse("40Mi"))
+			})
+
+			table.DescribeTable("should be migrated successfully, using guest agent on VM", func(mode v1.MigrationMode) {
```

Comment:
@vladikr Did you mean it like this? I was expecting only the post-copy case to be moved.

Reply:
Maybe we can create a test function so we can separate tests but still avoid duplication?

Something like:
```
testShouldBeMigratedSuccessfullyUsingGuestAgent := func(mode v1.MigrationMode) {
    // test code that's currently in DescribeTable
}
```

Then:
```
// in one place:
It("[test_id:2653] should be migrated successfully, using guest agent on VM with default migration configuration", func() {
    testShouldBeMigratedSuccessfullyUsingGuestAgent(v1.MigrationPreCopy)
})

// in another place
It("[QUARANTINE][test_id:5004] should be migrated successfully, using guest agent on VM with post copy", func() {
    testShouldBeMigratedSuccessfullyUsingGuestAgent(v1.MigrationPostCopy)
})
```

Maybe it's an overkill, I just feel bad completely duplicating the code..
WDYT? @vladikr @xpivarc

Reply:
@vladikr @xpivarc ping :)

Reply:
hmm, I was just suggesting moving the post-copy test out to its own context so it would have a separate initialization.
First, we need to make sure the test is passing correctly and then worry about optimizations.
That's my take ;)